### PR TITLE
Improve Homebrew for Ukrainian users

### PR DESCRIPTION
**Pull request by SMillerDev**
Updated translation of Ukrainian and more.

Thanks for the review @RuslanNedzelskyi

### CODE REVIEW

File: _data/locales/uk.yml [hunk 1]
```diff
@@ -30,4 +30,4 @@ uk:
       analytics: Дані для аналізу
     foot:
       code: Первинна розробка — <a href="https://mxcl.github.io/">Max Howell</a>.
-      page: Сайт — <a href="https://exomel.com/">Rémi Prévost</a>, <a href="https://mikemcquaid.com">Mike McQuaid</a> та <a href="https://cargocollective.com/danilalo">Danielle Lalonde</a>.
+      page: Сайт — <a href="https://exomel.com/">Rémi Prévost</a>, <a href="https://mikemcquaid.com">Mike McQuaid</a> та <a href="https://cargocollective.com/danilalo">Danielle Lalonde</a>.
```
Comment (mistydemeo) [inline _data/locales/uk.yml:33]: File should still end with a newline.

File: _data/locales/ru.yml [hunk 2]
```diff
@@ -1,33 +0,0 @@
-ru:
```
Comment (MikeMcQuaid) [inline _data/locales/ru.yml:1]: @SMillerDev Why is this deleted?

**Review by MikeMcQuaid — APPROVED** [review]
